### PR TITLE
Mal/user model

### PR DESCRIPTION
**Pull request by malwaredb86**
Created user class, templates for login and register pages, and authentication controller.

### CODE REVIEW

**Review by bahbond — APPROVED** [review]
Awesome Mal!

File: VenuApp/src/main/java/infinitycodecrew/VenuApp/controllers/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    }
+
+    @GetMapping("/logout")
+    public String logout(HttpServletRequest request) {
+        request.getSession().invalidate();
+        return "redirect:users/login";
+    }
+
+}
+
```
Comment (sammierosado) [inline VenuApp/src/main/java/infinitycodecrew/VenuApp/controllers/AuthenticationController.java:130]: Great work on the code! I just noticed a lot of empty lines being added in. This makes it harder to read the code. Can we be mindful of this in the future?

**Review by sammierosado — APPROVED** [review]

**Review by keydamm — APPROVED** [review]
Woooo!! Good job Mal :)

**Review by jaythemself — APPROVED** [review]
Looks great :)

**Pull request merged**